app/styles/ui/_diff.scss

```
@@ -269,13 +269,32 @@
 .CodeMirror-linebackground {
   &.diff-add {
     background: var(--diff-add-background-color);
-    border: var(--diff-add-border);
-    margin-top: 1px;
+    border-right: var(--diff-add-border);
+    border-left: var(--diff-add-border);
```

Similarly  to what we talked about re: high contrast theme borders being defined only for the high contrast theme I think we should do the same here. One approach  would be to add a mixin similar to https://github.com/desktop/desktop/blob/f9178938f1190dd289d62d4c36fb12f101c9c615/app/styles/mixins/_platform.scss#L9-L13

Then  we could do something like
```suggestion
    @include high-contrast-theme {
      border-right: var(--diff-add-border);
      border-left: var(--diff-add-border);
    }
```